The failures here are not related to this PR. They are all in `ucar.util.prefs.TestJavaUtilPreferences` (`testExport`, `testPutGet`, `testRemove`, `testRemoveNode`, `testSubnode`) all fail with:

```java
java.lang.AssertionError: Factory not set = java.util.prefs.FileSystemPreferences Property: ucar.util.prefs.PreferencesExtFactory
```

One thing I've noticed is that some of our test depend on the order in which they are executed, and the move to GitHub Actions has caused some of those to surface (not sure why though...different JDKs being used?). At any rate, this feels like it might be one of those cases (but I have not looked yet).

--- a/uibase/build.gradle
+++ b/uibase/build.gradle
@@ -29,3 +29,9 @@ dependencies {
     testCompile libraries["mockito"]
     testCompile libraries["truth"]
 }
+
+test {
+    // Tell java to use ucar.util.prefs.PreferencesExtFactory to generate preference objects
+    // Important for ucar.util.prefs.TestJavaUtilPreferences
+    systemProperty "java.util.prefs.PreferencesFactory", "ucar.util.prefs.PreferencesExtFactory"
+}

